testing/e2e/workload_test.go

```diff
@@ -45,9 +45,10 @@ var (
 )
 
 var (
-	pattern    = "\\\\U000[a-zA-Z0-9]+"
-	regexEmoji = regexp.MustCompile(pattern)
-	regexPod   = regexp.MustCompile("\\s*NAME\\s+READY\\s+STATUS\\s+RESTARTS\\s+AGE\\s*")
+	pattern       = "\\\\U000[a-zA-Z0-9]+"
+	regexEmoji    = regexp.MustCompile(pattern)
+	regexPod      = regexp.MustCompile("\\s*NAME\\s+READY\\s+STATUS\\s+RESTARTS\\s+AGE\\s*")
+	regexProgress = regexp.MustCompile(`[0-9]*\.*[0-9]*\s[a-zA-Z]+\s/\s[0-9]*[\.]*[0-9]*\s[a-zA-Z]+\s\[-+>*\]\s[0-9]*\.[0-9]+%.*`)
```

Comment:
lets refine the regex a bit to match better the progress output

Reply:
The regex is matching the progress output